doc/api/child_process.markdown

```diff
@@ -391,8 +391,28 @@ Doing so will cause the parent's event loop to not include the child in its
 reference count, allowing the parent to exit independently of the child, unless
 there is an established IPC channel between the child and parent.
 
-Example of detaching a long-running process and redirecting its output to a
-file:
+When using the `detached` option to start a long-running process, the process
+will not stay running in the background after the parent exits unless it is
+provided with a `stdio` configuration that is not connected to the parent.
+If the parent's `stdio` is inherited, the child will remain attached to the
+controlling terminal.
+
+Example of a long-running process, by detaching and also ignoring it's parent
+`stdio` file descriptors, in order to ignore the parents `SIGHUP` signal
```

Comment:
I think it's more important to mention the `unref()` of the child process than to discuss `SIGHUP`.